src/intake_sqlite/__init__.py

```diff
@@ -0,0 +1,26 @@
+"""A template repository for a Python package created by Catalyst Cooperative."""
+import logging
+
+import pkg_resources
+
+__version__: str = pkg_resources.get_distribution("intake-sqlite").version
+
+from intake_sqlite.sqlite_cat import *  # noqa: F403,F401
```

Comment:
Is this how imports are supposed to work?  In conjunction with the `__all__` definitions in the underlying modules?

Reply:
Yup that looks good to me.